app/frontend/components/domains/requirement-template/screens/edit-requirement-template-screen/index.tsx

```diff
@@ -122,57 +122,52 @@ export const EditRequirementTemplateScreen = observer(function EditRequirementTe
   const hasNoSections = watchedSectionsAttributes.length === 0
 
   return (
-    <RemoveScroll style={{ width: "100%", height: "calc(100% - 66px)", marginTop: "66px" }}>
-      <Flex flexDir={"column"} w={"full"} maxW={"full"} h="full" as="main">
-        <FormProvider {...formMethods}>
-          <EditableBuilderHeader requirementTemplate={requirementTemplate} />
-          <Flex flex={1} w={"full"} h={"1px"} borderTop={"1px solid"} borderColor={"border.base"}>
-            {isReorderMode ? (
-              <SectionsDnd sections={watchedSectionsAttributes} onCancel={closeReorderMode} onDone={onDndComplete} />
+    // the height 1px is needed other wise scroll does not work
+    // as it seems like the browser has issues calculating height for flex=1 containers
+    //<RemoveScroll style={{ width: "100%", height: "100%" }}>
```

Comment:
is this relevant? same Q for the other components with this comment - should remove or make the comment more clear since we aren't passing in 1px height anywhere anymore.